### PR TITLE
Create Back End boilerplate

### PR DESCRIPTION
**Pull request by kelpikz**

### CODE REVIEW

File: backend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#backend
+/venv
```
Comment (shb9019) [inline backend/.gitignore:2]: Why are you on windows?

Comment (shb9019) [reply backend/.gitignore:2]: Anyway, please add backward slash.

**Review by shb9019 — CHANGES_REQUESTED** [review]

File: backend/.gitignore (hunk 1 above)
Comment (kelpikz) [reply backend/.gitignore:2]: no I'm using linux

Comment (shb9019) [reply backend/.gitignore:2]: Wait. Sorry. I got confused. Never mind.

Comment (kelpikz) [reply backend/.gitignore:2]: what shd i add backslash for?

File: backend/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+pathspec==0.8.0
+pylint==2.6.0
+regex==2020.10.15
+six==1.15.0
+SQLAlchemy==1.3.20
+toml==0.10.1
+typed-ast==1.4.1
+typing-extensions==3.7.4.3
+Werkzeug==1.0.1
+wrapt==1.12.1
```
Comment (shb9019) [inline backend/requirements.txt]: Are we using this many packages already?

Comment (shb9019) [reply backend/requirements.txt]: If not, can we trim down to the bare essentials?

**Review by shb9019 — APPROVED** [review]

**Pull request merged**